README.md

```diff
@@ -7,7 +7,8 @@ Confidential Containers is an open source community working to leverage
 to protect containers and data and to deliver cloud native 
 confidential computing.
 
-**Release v0.1.0 is Live!** See [Release Notes](./releases/v0.1.0.md) or [Quickstart Guide](./quickstart.md)
+**We have a new release every 6 weeks!**
+See [Release Notes](./releases/) or [Quickstart Guide](./quickstart.md)
```

Comment:
If the intent is to render the above text in a newline, then it's not happening.